OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/Optimobile.java

```diff
@@ -331,6 +335,20 @@ public static void pushTokenStore(@NonNull Context context, @NonNull final PushT
         trackEvent(context, AnalyticsContract.EVENT_TYPE_PUSH_DEVICE_REGISTERED, props, System.currentTimeMillis(), true);
     }
 
+
+    public static void notificationEnablementStatusChanged(@NonNull Context context, boolean notificationsEnabled) {
+        JSONObject props = new JSONObject();
+
+        try {
+            props.put("notificationsEnabled", notificationsEnabled);
+        } catch (JSONException e) {
+            e.printStackTrace();
+            return;
+        }
+
+        trackEvent(context, AnalyticsContract.EVENT_TYPE_PUSH_NOTIFICATION_ENABLEMENT_CHANGED, props, System.currentTimeMillis(), true);
```

Comment:
can call trackEventImmediately? it has those 2 last args set

Reply:
Done